src/System.Runtime.WindowsRuntime/src/System/IO/StreamOperationAsyncResult.cs

```diff
@@ -150,7 +150,12 @@ internal bool CancelStreamOperation()
             if (_callbackInvoked)
                 return false;
 
-            _asyncStreamOperation.Cancel();
+            try
+            {
+                _asyncStreamOperation?.Cancel();
+            }
+            catch { }
```

Comment:
NullReferenceException on that field.
Would you like me to remove the try-catch? I've only added it as the close method also do this, and it's been called from the finalizer and it will crash the process.
IMO in this case, eating the exception will no do harm as those classed are about to be reclaimed, and there is no other thing for us to do in case of an exception.

Would you like me to remove the try-catch block?